packages/front-end/package.json

```diff
@@ -6,6 +6,7 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
+    "plop": "plop",
```

Comment:
Adds tooling for code generation so we can do `yarn plop` and generate a new component. This command works only in the `front-end` package.

```
➜  front-end git:(event-webhooks-ui-view) ✗ yarn plop
yarn run v1.22.15
$ plop
? What should the name of the component be? MyTestComponent
✔  ++ /components/MyTestComponent/MyTestComponent.tsx
✔  ++ /components/MyTestComponent/MyTestComponent.stories.tsx
✨  Done in 30.36s.
```

Reply:
The templates need to be in one central spot for plop to find them. It could get a bit messy if all of them are in the same directory. Also we'd have to dig down into the packages when creating files. If you find this helpful, you can add the same version to the backend and create templates in the same way. As long as it's the same version, it shouldn't cause any additional lines added to the lock file.

Reply:
An update on this, I opened a PR that adds plop and uses it for backend code generation: https://github.com/growthbook/growthbook/pull/797 – You can review whenever you get a free moment.

I think once both are merged, we can worry about DRY'ing up the plopfile and templates.

Reply:
Works for me